docs - nginx client body size limitation (#3222)
treeverse/lakeFS: docs/deploy/k8s.md

Options to do so include a Kubernetes Service of type `LoadBalancer`, or a Kubernetes Ingress.
By default, lakeFS operates on port 8000, and exposes a `/_health` endpoint which you can use for health checks.

> Nginx Ingress Controller has a default client's body size of 1 MB.

Reviewer: ```suggestion
The NGINX Ingress Controller by default limits the client body size to 1 MiB.
```

But I would consider calling it out more explicitly, e.g.
```suggestion
The NGINX Ingress Controller by default limits the client body size to 1 MiB.  This is not useful for working with large objects, and should be increased at least to 5 MiB to allow multipart uploads, and to much more for safe usage.
```
